coalib/bearlib/aspects/Redundancy.py

```diff
@@ -0,0 +1,247 @@
+from coalib.bearlib.aspects import (
+    Aspect, AspectDocumentation, AspectSettng, Root)
+
+
+Root.new_subaspect(
+    'Redundancy',
+    doc=AspectDocumentation(
+            definition="This aspect describes redundancy in your source code.",
+            cause="Redundancies are often caused by forgetting to remove "
+                  "code during refactorings.",
+            importance_reason="Redundant code makes your code harder to read
+                              "and understand.",
+            fix_suggestions="Redundant code can usually be removed without "
+                            "consequences."
+        )
+    )
+
+Root.Redundancy.new_subaspect(
+    'Clone',
+    doc=AspectDocumentation(
+            definition="""
+            Code clones are multiple pieces of source code in your
+            codebase that are very similar.
+            """,
+            cause="""
+            Clones usually result out of copy and pasting.
+            """,
+            example_language='python',
+            importance_reason="""
+            Small differences between clones can be indications of missed fault
+            fixes leading to the hypothesis that such clones are related to
+            faults.
```

Comment:
I find this hard to understand, also there's more reasons: it's bad for maintainability, blows up code and what you mean seems to be that bugs may appear in both versions just fixed in one or so